node/src/protocol/cryptography.rs

```diff
@@ -163,6 +163,23 @@ impl CryptographicProtocol for ResharingState {
     }
 }
 
+#[async_trait]
+impl CryptographicProtocol for RunningState {
+    async fn progress<C: CryptographicCtx + Send + Sync>(
+        mut self,
+        ctx: C,
+    ) -> Result<NodeState, CryptographicError> {
+        if self.triple_manager.potential_len() < 2 {
+            self.triple_manager.generate();
+        }
```

Comment:
so, we're generating more triples on the fly when we run low? Won't this be very computationally expensive or since we're just generating one, it will be fine? But wouldn't that still impose a good amount of latency with all the messaging the triple generation protocol requires?

Reply:
So this is just pretty much a placeholder. We need to implement `TripleStockpile` that operates on top of `TripleManager` and actively tries to initiate generation when it can (the ticket is in the epic, but hasn't been scoped out yet).